src/components/deck/default-deck.js

```diff
@@ -62,10 +62,23 @@ export default function DefaultDeck({
     [overviewMode, toggleMode]
   );
 
+  const onMobileSlide = e => {
+    if (navigator.maxTouchPoints < 1) return;
+    switch (e.dir) {
+      case 'Left':
+        deck.current.advanceSlide();
```

Comment:
Let's actually swap this out for `stepForward` so we ensure Appear-items also show up when they swipe to advance. A good slide to test this is the JS-demo's `http://localhost:3000/?slideIndex=10&stepIndex=0`